configure

```diff
@@ -2187,14 +2187,35 @@ check_higgstools_libs() {
             return 1
         fi
     else
+        local filename
+        filename=$(basename -- "$found_lib")
+        local extension="${filename##*.}"
         local HIGGSTOOLSLIBS_="-lHiggsTools"
-        if [ -z "$higgstools_lib_dir" ]; then
-            # assume that the linker will look in the default paths
-            HIGGSTOOLSLIBS="$HIGGSTOOLSLIBS_"
-        else
-            # assume that the linker will look in the default paths
-            HIGGSTOOLSLIBS="-L$higgstools_lib_dir -Wl,-rpath,$higgstools_lib_dir $HIGGSTOOLSLIBS_"
-        fi
+        if [ "$extension" = "a" ] ; then
+            if [ -z "$higgstools_lib_dir" ]; then
+                # assume that the linker will look in the default paths
+                HIGGSTOOLSLIBS="-Wl,-Bstatic -$HIGGSTOOLSLIBS_ -Wl,-Bdynamic"
+            else
+                # assume that the linker will look in the default paths
+                HIGGSTOOLSLIBS="-L$higgstools_lib_dir -Wl,-rpath,$higgstools_lib_dir,-Bstatic $HIGGSTOOLSLIBS_ -Wl,-Bdynamic"
+            fi
+	    elif [ "$extension" = "so" ] ; then
```

Comment:
I think there is something wrong with the indentation of this line. :)

Reply:
Sorry, I was editing this on a different PC where vim (the one true editor) was not set to keep spaces and tabs consistent.

Reply:
Thanks for fixing this! (But let's discuss the issue with the one and only true editor again! ;) )